api/types.proto

```diff
@@ -489,12 +489,17 @@ message NetworkAttachmentConfig {
 	// Target specifies the target network for attachment. This value must be a
 	// network ID.
 	string target = 1;
+	
 	// Aliases specifies a list of discoverable alternate names for the service on this Target.
 	repeated string aliases = 2;
+	
 	// Addresses specifies a list of ipv4 and ipv6 addresses
 	// preferred. If these addresses are not available then the
 	// attachment might fail.
 	repeated string addresses = 3;
+	
+	//Driver options is a map of driver specific options for the network target
```

Comment:
@abhinandanpb as a best practice always add the dependent PR links and that will help the reviewers to map the usage.

@stevvooe @aaronlehmann I think the usage details as expressed in https://github.com/moby/moby/pull/33130 and https://github.com/docker/cli/pull/62 will help explain this new option. Essentially, these are some of the user-specified service options that are passed to the network drivers. Labels are NOT sent to the drivers and in other docker objects we have introduced driver-opts to pass on such operator specific params to the drivers (volumes and network). Service object was missing it and via this PR @abhinandanpb is introducing it.